src/lua/lua_cryptobox.c

```diff
@@ -1438,21 +1486,25 @@ lua_cryptobox_hash_finish(struct rspamd_lua_cryptobox_hash *h)
 {
 	uint64_t ll;
 	unsigned char out[rspamd_cryptobox_HASHBYTES];
-	unsigned int ssl_outlen = sizeof(out);
+	size_t ssl_outlen = sizeof(out);
 
 	switch (h->type) {
 	case LUA_CRYPTOBOX_HASH_BLAKE2:
 		rspamd_cryptobox_hash_final(h->content.h, out);
 		memcpy(h->out, out, sizeof(out));
 		break;
 	case LUA_CRYPTOBOX_HASH_SSL:
-		EVP_DigestFinal_ex(h->content.c, out, &ssl_outlen);
+		EVP_DigestFinal_ex(h->content.c, out, (unsigned int *) &ssl_outlen);
```

Comment:
We cannot cast pointer of `size_t*` to `unsigned int*`, as it will likely be corrupted. Please use a dedicated `unsigned int` variable. I know it's annoying but I see no other options.